project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.2.8
+sbt.version=1.3.10
```

Comment:
This PR adds the `datarepo-client` dependency. That has a transitive dependency on `javax.ws.rs:javax.ws.rs-api:2.1.1`. There is a bug in sbt 1.2 - actually in Ivy - that causes that transitive dependency to fail; see https://github.com/sbt/sbt/issues/3618.

Upgrading to sbt 1.3 addresses this problem, because sbt 1.3 uses Coursier, not Ivy. Alternate solution is to ask the data repo team to change their dependency, but the sbt 1.3 upgrade feels worthwhile anyway.

Note that broadinstitute/firecloud-develop#2138 enables coursier caching for local-environment servers. It's not technically required but it avoids re-downloading dependencies every time you start your local server.